cmd/emulator/Dockerfile

```diff
@@ -12,16 +12,28 @@ WORKDIR /app
 # add the pubkey of github.com to knownhosts, so ssh-agent doesn't bark
 RUN mkdir -p /root/.ssh && ssh-keyscan -t rsa github.com >> /root/.ssh/known_hosts
 RUN git config --global 'url.ssh://git@github.com/.insteadOf' https://github.com/
+RUN apt-get update
+RUN apt-get -y install apt-utils gcc-arm-linux-gnueabi
 
 COPY . .
 
 ARG TARGETOS 
 ARG TARGETARCH
-RUN --mount=type=ssh \
+RUN if [ "$TARGETARCH" = "arm64" ] ; then \
```

Comment:
@sideninja or @turbolent, any chance you know how to add branching in a Dockerfile? I would like to set a different flag `CC=...` in the `go build` command depending on `TARGETARCH`, tried a few options but all were unsuccessful.

Reply:
update: no need to look at it for now, I got help and I'm investigating the next errors.